sapp/ui/frontend/src/Filter.js

```diff
@@ -746,12 +751,15 @@ const FilterForm = (props: {
   );
 };
 
-const SaveFilterModal = (
+const SaveAndImportFilterModals = (
   props: $ReadOnly<{|
     currentFilter: FilterDescription,
-    visible: boolean,
-    hide: void => void,
-    onSave: FilterDescription => void,
+    setCurrentFilter: FilterDescription => void,
+    saveModalVisible: boolean,
```

Comment:
nit: I think by convention we use `is` as prefix for boolean flags (disclaimer: I'm highly unsure about whether that is actually a thing)

Reply:
I tried searching for `isBoolean` and `isboolean` in the repo, but couldn't find it :/ are you sure that it was what you meant 🙃

Also, we could defy convention and make this `isBoolean` if you really want, or we could do away with the type declaration all together!